### DISCUSSION TITLE
No games showing

### DISCUSSION THREAD

**Discussion by jonmunson** [OnionUI/Onion#1146] [Support] [answered]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

v4.2.0-RC-snapshot-a5062407

### Miyoo Firmware Version

202305052130

### Provide a clear and concise description of the issue

On a fresh install, no games show in the Games section. Games are loaded into the Roms folders, and work on stock OS.

### Steps to Reproduce

1. Install fresh Onion OS
2. Load roms and bios files
3. Refresh roms
4. No games visibile

### Provide any additional context or information that may help us investigate your issue

Roms all tested and working. Also tried with clean install and Tiny Best Set Go, as known roms and bios files. 
Games are able to load and play directly in RetroArch.

**Comment by Aemiii91** [comment, marked as answer]
Try enabling the systems in Package Manager. If that doesn't do anything, you'll need to do a reinstall.

**Reply by jonmunson** [reply]
This worked! I enabled all all systems and refreshed the roms, all games show now.

This step would be worth adding to the documentation, as none of the systems were enabled from a fresh install.

**Reply by Aemiii91** [reply]
I agree, it's already on my to-do 😉
